addons/mail/static/src/new/attachment_viewer/attachment_viewer.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.attachment_viewer" owl="1">
+        <div class="o-mail-attachment-viewer flex-column align-items-center d-flex w-100 h-100" tabindex="0">
+            <div class="o-mail-attachment-viewer-header d-flex w-100 bg-black-75 text-400">
+                <div t-if="isViewable" class="o-mail-attachment-viewer-headerItem d-flex align-items-center ms-4 me-2">
+                    <i t-if="state.attachment.isImage" class="fa fa-picture-o" role="img" title="Image"/>
+                    <i t-if="state.attachment.isPdf" class="fa fa-file-text" role="img" title="PDF file"/>
+                    <i t-if="state.attachment.isText" class="fa fa-file-text" role="img" title="Text file"/>
+                    <i t-if="state.attachment.isVideo" class="fa fa-video-camera" role="img" title="Video"/>
+                </div>
+                <div class="o-mail-attachment-viewer-headerItem o-mail-attachment-viewer-name d-flex align-items-center mx-2">
+                    <span class="o-mail-attachment-viewer-nameText text-truncate" t-out="state.attachment.displayName"/>
+                </div>
+                <div class="flex-grow-1"/>
+                <div class="o-mail-attachment-viewer-buttonDownload o-mail-attachment-viewer-headerItem o-mail-attachment-viewer-headerItemButton d-flex align-items-center px-3 cursor-pointer" role="button" title="Download">
+                    <a t-att-href="state.attachment.downloadUrl" class="text-reset" download>
+                        <i class="o-mail-attachment-viewer-headerItemButtonIcon fa fa-download fa-fw" role="img"/>
+                        <span>Download</span>
+                    </a>
+                </div>
```

Comment:
Whole `div` should do download (so icon + span). `cursor-pointer` is correctly placed